build adolc.bat

```diff
@@ -0,0 +1,6 @@
+SET currentdir=%cd%
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsDevCmd.bat"
+call "C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\Tools\VsMSBuildCmd.bat"
+cd %currentdir%
+set INCLUDE="%INCLUDE%C:\Users\Zak Smith\Downloads\boost_1_67_0;C:\ZS\autodiff\submodules\adol-c\MSVisualStudio\v14\nosparse;"
```

Comment:
Can you add a TODO comment wherever there are hard-coded paths.   We can fix them later, but it may be useful to flag them here, as e.g. the Visual Studio ones will be the same on many machines but not all.